vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -159,6 +165,28 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
         return html.Div([dashboard_title, settings, page_title, nav_bar, nav_panel, control_panel, components])
 
     def _arrange_page_divs(self, page_divs: _PageDivsType):
+        collapsable_icon = html.Div(
+            children=[
+                dmc.Tooltip(
+                    id="collapse_tooltip",
+                    label="Hide Menu",
+                    offset=20,
+                    withArrow=True,
+                    children=[
+                        html.Span(
+                            "keyboard_double_arrow_right",
+                            className="material-symbols-outlined",
+                            id="collapse_icon",
+                        )
+                    ],
+                    position="right-end",
+                    arrowOffset=10,
+                    style={"marginTop": "12px", "marginLeft": "8px"},
+                )
+            ],
+            className="collapsable-div",
+        )
```

Comment:
I would suggest the following changes - some will make more sense when we add the relevant CSS:

- Remove the outer div, as the outer div is not necessary. Instead we should leverage [absolute/relative positioning](https://www.geeksforgeeks.org/how-to-set-position-absolute-but-relative-to-parent-in-css/) and the parent element should be the `left-side`
- Remove the brackets around the html.Span 
- Change position to right
- Remove inline-style (To achieve what you wanted to achieve, we don't need the inline style, but instead need to adapt the height of the target element. The dmc.Tooltip will by default be positioned correctly then) -> I'll explain later in the custom css changes
- Add `className="collapse-button-tooltip"` (we need this later to update the height or the target element) - optimally I would like to do that only via the ID, but somehow this doesn't work with the mantine component. So had to add the className here. But maybe try and see if we could also do this only via the `id=collapse_tooltip` instead of adding a className here

```suggestion
        collapsable_icon = dmc.Tooltip(
            html.Span("keyboard_double_arrow_right", className="material-symbols-outlined", id="collapse_icon"),
            id="collapse_tooltip",
            label="Hide Menu",
            offset=24,
            withArrow=True,
            position="right",
            arrowOffset=10,
            className="collapse-button-tooltip",
        )
```

Reply:
All suggestions are included, but the last one as targeting by ID didn't work.

Reply:
Thanks for checking!